src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -32,21 +32,22 @@ class Tribe__Events__Linked_Posts__Chooser_Meta_Box {
 	protected $singular_name;
 
 	public function __construct( $event = null, $post_type = null ) {
-		$this->tribe        = Tribe__Events__Main::instance();
-		$this->linked_posts = Tribe__Events__Linked_Posts::instance();
-		$this->post_type = $post_type;
-		$this->singular_name = $this->linked_posts->linked_post_types[ $this->post_type ]['singular_name'];
+		$this->tribe                   = Tribe__Events__Main::instance();
+		$this->linked_posts            = Tribe__Events__Linked_Posts::instance();
+		$this->post_type               = $post_type;
+		$this->singular_name           = $this->linked_posts->linked_post_types[ $this->post_type ]['singular_name'];
 		$this->singular_name_lowercase = $this->linked_posts->linked_post_types[ $this->post_type ]['singular_name_lowercase'];
 		$this->get_event( $event );
 
 		add_action( 'wp', array( $this, 'sticky_form_data' ), 50 ); // Later than events-admin.js itself is enqueued
 	}
 
 	/**
-	 * Work with the specifed event object or else use a placeholder if we are in
-	 * the middle of creating a new event.
+	 * Work with the specifed event object or else use a placeholder if in the middle of creating a new event.
 	 *
-	 * @param null $event
+	 * @since ??
```

Comment:
Here are elsewhere: what is your plan for updating the `??` placeholders? This isn't something the release dev will have time for. If you are unsure and didn't modify the function in any case, I'd suggest stripping it out.